README.md

```diff
@@ -21,26 +21,61 @@ This repository provides solutions for configuring [LaunchDarkly code references
-| `commitUrlTemplate` | If provided, LaunchDarkly will attempt to generate links to your Git service provider per commit. Example: `https://github.com/launchdarkly/git-flag-parser/commit/${sha}`. Allowed template variables: `branchName`, `sha`. If `commitUrlTemplate` is not provided, but `repoUrl` is provided, LaunchDarkly will automatically generate links for github or bitbucket repo types. | "" | no |
-| `hunkUrlTemplate` | If provided, LaunchDarkly will attempt to generate links to your Git service provider per code reference. Example: `https://github.com/launchdarkly/git-flag-parser/blob/${sha}/${filePath}#L${lineNumber}`. Allowed template variables: `sha`, `filePath`, `lineNumber`. If `hunkUrlTemplate` is not provided, but `repoUrl` is provided, LaunchDarkly will automatically generate links for github or bitbucket repo types. | "" | no |
+macOS users should download the darwin release for their respective system architecture.
+
+The `git-flag-parser` requires [The Silver Searcher](https://github.com/ggreer/the_silver_searcher#installing) to be installed as a dependency, so make sure it has been installed and added to your system PATH before running the git-flag-parser.
+
+A number of command-line arguments are available to the parser, some optional, and some required. Command line arguments may be passed to the program in any order.
+
+### Required arguments
+
+| Option | Description |
+|-|-|
+| `accessToken` | LaunchDarkly [personal access token](https://docs.launchdarkly.com/docs/api-access-tokens) with writer-level access, or access to the `code-references` [custom role](https://docs.launchdarkly.com/v2.0/docs/custom-roles) resource |
+| `dir` | Path to existing checkout of the git repo. If a cloneEndpoint is provided, this option is not required. |
+| `projKey` | A LaunchDarkly project key. |
+| `repoName` | Git repo name. Will be displayed in LaunchDarkly |
 
-## Testing
+Here's an example shell invocation of the git-flag-parser for one of LaunchDarkly's demo repositories:, with the binary located in the current directory, and a minimal configuration:
+
+```shell
+./git-flag-parser \
+  -accessToken="$YOUR_LAUNCHDARKLY_ACCESS_TOKEN" \
+  -dir="path/to/git/repo" \
+  -repoName="my-repository" \
+  -projKey="default"
+```
+
+### Optional arguments
+
+| Option | Description | Default |
+|-|-|-|
+| `baseUri` | Set the base URL of the LaunchDarkly server for this configuration. Only necessary if using a private instance of LaunchDarkly. | `https://app.launchdarkly.com` |
+| `cloneEndpoint` | If provided, will clone the repo from this endpoint to the provided `dir`. If authentication is required, this endpoint should be authenticated. Supports the https protocol for git cloning. Example: `https://username:password@github.com/username/repository.git` | n/a |
```

Comment:
Might as well remove this now right?

Reply:
Remove with https://github.com/launchdarkly/git-flag-parser/pull/33